src/database/models/course-question-assessment-info.ts

```diff
@@ -79,7 +81,13 @@ CourseQuestionAssessmentInfo.init({
         type: DataTypes.JSONB,
         allowNull: true,
         defaultValue: null,
-    }
+    },
+    originatingQuestionAssessmentInfo: {
```

Comment:
Missing association, even if we don't use it we should add it so auto-generated db get's fk